modules/postgres/postgres.go

```diff
@@ -35,12 +37,8 @@ func (c *PostgresContainer) ConnectionString(ctx context.Context, args ...string
 		return "", err
 	}
 
-	extraArgs := ""
-	for _, arg := range args {
-		extraArgs += " " + arg
-	}
-
-	connStr := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s %s", host, containerPort.Port(), c.user, c.password, c.dbName, extraArgs)
+	extraArgs := strings.Join(args, "&")
+	connStr := fmt.Sprintf("postgres://%s:%s@%s/%s?%s", c.user, c.password, net.JoinHostPort(host, containerPort.Port()), c.dbName, extraArgs)
```

Comment:
I'd like to double check with you: this connection string format will be compatible with pg, right?

Reply:
I said nothing: I understood PG as the Go client for Postgres, not Postgres itself 🙈